[docs] Add a note about the new dry run option (#6973)
apollographql/router: docs/source/routing/performance/caching/in-memory.mdx

Precomputed plans will be cached before the router switches traffic over to the new schema.

> <Tip>
> 
> You can also send the header `Apollo-Expose-Query-Plan: dry-run` for [generating query plans at runtime](https://www.apollographql.com/docs/graphos/reference/federation/query-plans#outputting-query-plans-with-headers) which can be used to warm up your cache instances with a custom defined operation list.
> 
> </Tip>

Reviewer: Readability suggestion:

```suggestion
<Tip>

You can also send the `Apollo-Expose-Query-Plan: dry-run` header to [generate query plans at runtime](/graphos/reference/federation/query-plans#outputting-query-plans-with-headers). This is useful for warming up your cache instances with a custom operation list.

</Tip>
```
